test/utils/test-utils.js

```diff
@@ -190,9 +190,19 @@ async function runInstall(cwd) {
     });
 }
 
+const runServe = (args, testPath) => {
```

Comment:
I would say the former implementation was good. Why should `testPath` be passed in  explcitly?

Reply:
Because runServe needs to be shared, and needs the path of the test to run

Reply:
Yeah, makes sense 👍